changelogs/current.yaml

```diff
@@ -15,6 +15,12 @@ behavior_changes:
     When ``append_x_forwarded_host`` is enabled for a given route action it is now only appended iff it is different from the last
     value in the list. This resolves issues where a retry caused the same value to be appended multiple times. This
     behavioral change can be temporarily reverted by setting runtime guard ``envoy_reloadable_features_append_xfh_idempotent`` to false.
+- area: ext_proc
+  change: |
+    ext_proc filter uses hard-coded header mutation rule to handle header mutations in the ImmediateResponse message from the
```

Comment:
Suggestion. Replace first 3 sentences to "Apply header mutation rules from the ext_proc config to the ImmediateResponse."

Reply:
Done